Dockerfile

```diff
@@ -3,26 +3,20 @@ MAINTAINER Tim Perry <pimterry@gmail.com>
 
 USER root
 
-RUN apt-get update && \
-    export DEBIAN_FRONTEND=noninteractive && \
-    # Set password to temp-password - reset to random password on startup
-    echo mysql-server mysql-server/root_password password temp-password | debconf-set-selections && \
-    echo mysql-server mysql-server/root_password_again password temp-password | debconf-set-selections && \   
-    # Instal MySQL for data, node as the JS engine for uglifier
-    apt-get install -y mysql-server nodejs
-    
-COPY . /opt/staytus
+ENV DEBIAN_FRONTEND="noninteractive"
```

Comment:
A bit debatable, but it's slightly nicer to keep this inside the RUN statement.

Having it in the RUN scopes it to just exist for that command (see http://stackoverflow.com/questions/33379393/docker-env-vs-run-export). Nothing else needs this var, and we certainly don't need to have it set in the resulting Docker image. Doesn't cost much obviously, but it's a bit tidier without.